package.json

```diff
@@ -14,6 +14,7 @@
     "react-router-dom": "^5.2.0",
     "react-scripts": "4.0.2",
     "react-slick": "^0.28.0",
+    "react-toastify": "^7.0.3",
```

Comment:
I went through the Snackbars that are available in Material UI, but it seems that they cannot be made HOCs and also needs to be managed where ever needed.

Reply:
> they cannot be made HOCs
Why HOC for this though? 🤔

Create a component and wrap around Snackbar, keep the state in that component 🤔

This would reduce bundle size, both JS and of CSS.

Reply:
Removed toast library, for now, we don't need it anymore, and yes do you prefer maintaining multiple states over bundle size, that's too much redundant code.

Reply:
> do you prefer maintaining multiple states over bundle size

Yes. But no, we'd not have multiple states. I am not sure what you have in mind. Like I said, Wrap a component around Snackbar, and manage the state in it. That's pretty much how that library does too. They have to maintain state SOMEWHERE.